Xamarin.Forms.Platform.UAP/MasterDetailControl.cs

```diff
@@ -22,6 +22,9 @@ public class MasterDetailControl : Control, IToolbarProvider
 		public static readonly DependencyProperty ShouldShowSplitModeProperty = DependencyProperty.Register(nameof(ShouldShowSplitMode), typeof(bool), typeof(MasterDetailControl),
 			new PropertyMetadata(default(bool), OnShouldShowSplitModeChanged));
 
+		public static readonly DependencyProperty ShouldShowNavigationBarProperty = DependencyProperty.Register(nameof(ShouldShowNavigationBar), typeof(bool), typeof(MasterDetailControl),
+		new PropertyMetadata(true, OnShouldShowSplitModeChanged));
```

Comment:
Does this need to run `OnShouldShowSplitModeChanged`, or is that a copy paste error?

Reply:
Ño it should! since the logic is there

Reply:
Just checking! :D